File: app/src/main/java/app/deadmc/devnetworktool/models/SimpleString.java
package app.deadmc.devnetworktool.models;

public class SimpleString {

    String text;
    boolean bold;

    public SimpleString(String text) {
        this.text = text;
        this.bold = false;
    }

    public SimpleString(String text,boolean bold) {
        this.text = text;
        this.bold = bold;
    }

    public String getText() {
        return text;
    }

    public boolean isBold() {
        return bold;
    }


}

File: app/src/main/java/app/deadmc/devnetworktool/data/models/SimpleString.kt
package app.deadmc.devnetworktool.data.models

class SimpleString {

    var text: String
    var isBold: Boolean = false

    constructor(text: String) {
        this.text = text
    }

    constructor(text: String, bold: Boolean) {
        this.text = text
        this.isBold = bold
    }


}
